### PR TITLE
Modify CaloLayer1 DQM for 5BX needed for Ecal

### PR DESCRIPTION
**Pull request by hftsoi**
#### PR description:

Modifications are made to add DQM CaloLayer1 monitoring elements for 5BX, which are needed for ECAL: two 5-bin 1D plots for TPs averages vs BX (with or without Et-weighted), and five 2D TPs occupancy plots. Unpacker is modified to point to the proper sample when filling the 5BX histograms (the middle BX is the "standard").

#### PR validation:

For testing, the unpacker is modified to fill 5 times the EcalDigisBx for 5BX inside the "if(N_BX==1)" since now we always get BX=1, so the 5BX histograms can get filled for testing (this modification is for our own testing purpose so is not included in this PR). The codes can be compiled and are tested with an offline DQM workflow "runTheMatrix.py -l 10823.0", the 5BX histograms show up in the DQM root files and are properly filled.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Before submitting your pull requests, make sure you followed this checklist:
- verify that the PR is really intended for the chosen branch
- verify that changes follow [CMS Naming, Coding, And Style Rules](http://cms-sw.github.io/cms_coding_rules.html)
- verify that the PR passes the basic test procedure suggested in the [CMSSW PR instructions](https://cms-sw.github.io/PRWorkflow.html)

### CODE REVIEW

**Comment by cecilecaillol** [comment]
please test

File: EventFilter/L1TRawToDigi/plugins/implementations_stage2/CaloLayer1Unpacker.cc [hunk 1]
```diff
@@ -32,6 +32,7 @@ namespace l1t {
         makeRegions(ctp7_phi, ctp7Data, res->getRegions());
       } else if (N_BX == 5) {
         const uint32_t* ptr5 = ptr;
+        ptr += 192 * 2;
```
Comment (aloeliger) [inline EventFilter/L1TRawToDigi/plugins/implementations_stage2/CaloLayer1Unpacker.cc:35]: Is this change endorsed by @asavincms ?

**Comment by jfernan2** [comment]
Dear @hftsoi 
Can you please add yourself along with your github username to corresponding e-group here?
https://twiki.cern.ch/twiki/bin/viewauth/CMS/DQMContacts#ECAL
Thanks

**Comment by asavincms** [comment]
yes, this is ok

On Apr 23, 2021, at 12:45 PM, aloeliger ***@***.******@***.***>> wrote:



@aloeliger commented on this pull request.

________________________________

In EventFilter/L1TRawToDigi/plugins/implementations_stage2/CaloLayer1Unpacker.cc<https://github.com/cms-sw/cmssw/pull/33510#discussion_r619123640>:

> @@ -32,6 +32,7 @@ namespace l1t {
         makeRegions(ctp7_phi, ctp7Data, res->getRegions());
       } else if (N_BX == 5) {
         const uint32_t* ptr5 = ptr;
+        ptr += 192 * 2;


Is this change endorsed by @asavincms<https://github.com/asavincms> ?

**Comment by cecilecaillol** [comment]
+l1

**Comment by jfernan2** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**